### ISSUE TITLE
As an OEC Provider with Multiple Organizations, I can see a page with all organizations

### ISSUE THREAD

**Issue by Derekskylight** [ctoec/data-collection#1348]
## Background
As an OEC Provider with Multiple Organizations (>1), I should be able to view the rosters of my different organizations. This will provide more clarity for these specific users in order to view their enrollment data.

## Acceptance Criteria
- **Landing Page**
   - Upon log-in, the Provider is brought onto a landing page specific to themself:
   - 'Organizations' is placed in header text format
   - Link to each of the organization's roster page is shown horizontally 
   - Breakdown of number of _Infant/Toddler_, _Preschool_, _School Aged_ children are shown directly below each organization
- **Roster Page**
   - Once a specific organization is clicked, the user is redirected to that organization's roster page
   - Link for _'Back to all organizations'_ is shown directly below navigation, linking back to landing page

## Designs
[Full Figma files can be found here.](https://www.figma.com/file/GcYwZ05CjnG6QZNYlsCS1X/ECE-Reporter-%E2%80%93-Final-Designs?node-id=86%3A348)

#### Landing Page
![Multi-Org.PNG](https://images.zenhubusercontent.com/602d78337403e44d7db5c4f3/682a6e51-0d0f-496c-9285-24e62299e58a)

#### Roster Page for Multi-Org Providers
![Multi-Org2.PNG](https://images.zenhubusercontent.com/602d78337403e44d7db5c4f3/e6e8efc5-024e-4f0d-8f27-07758031b789)

**Comment by JordanGuinn** [comment]
@ehuang525 @Derekskylight As is captured in #1275, we have at least one user who has permission to access multiple organizations, but should only actually have visibility into a subset of those sites in one of those organizations.  Do we simply want to update the various data points on the multi-org page (and the roster, for that matter) to reflect those permissions?  Or do we need to explicitly point that subset of sites to the user?

The pieces of data that I see as immediately being potentially impacted are:
- Child counts by age group on the multi-org view
- List of sites user can filter by in the roster
- List of enrollments included in the roster

**Comment by ehuang525** [comment]
Hm, good question. If this user only has access to specific sites within multiple organizations, these data points should probably reflect what they have access to. I'm assuming it'd be confusing otherwise. I'm for updating data points to reflect those permissions but would want confirmation from @jlhogan that this is the right move and that we're not missing anything.

**Comment by jlhogan** [comment]
yes, confirm i agree with emily. e.g. organiztion A exists with sites One, Two and Three, with 5 students each. User has access to only sites One and Two, when they click on Organization A they see that there are 10 students across 2 sites.

**Comment by Derekskylight** [comment]
In the roster page, there is a description including the number of sites for an organization underlined ("Your roster has 74 children across 8 sites"). Could it be possible to link that number of sites to this page once developed?

**Comment by JordanGuinn** [comment]
They should already have a `Back to all organizations` button as part of the designs for this ticket, no?  Besides, that site count only pertains to the organization they're looking at, so it wouldn't make sense for that to link back to their multi-org list.

**Comment by ehuang525** [comment]
@Derekskylight I don't think we need to link those numbers. I think Deana was just pointing out today that making static information underlined implies there is interactivity -- but since interactivity is something we don't want, we can think about how to design the styling of the text to match its intended purpose.
